mantidproject/mantid
# Add ParaView libraries to the PYTHONPATH.

Description of work.

This adds the necessary directories to `PYTHONPATH` so that a user can easily import ParaView's python module. My initial goal is to add a small test button the settings to confirm that the OpenGL configuration is sufficient for ParaView. In the future, this could be an additional ParaView interface used alongside the VSI.

**To test:**


1. From within MantidPlot, make the sure the following command complete successfully and returns the correct version.

``` python
from paraview.simple import *
GetParaViewVersion()
```

This has been added as a MantidPlot unit test, but paths for the binary package are not automatically checked.
1. From mantidpython, make the sure the following command completes successfully and returns the correct version. 

``` python
from paraview.simple import *
GetParaViewVersion()
```

This has been added as a PythonInterface unit test and as a system test. Note that the pull requests on test the system test of RHEL7.

There is no GitHub issue associated with this pull request.

Still need to add release notes.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
